src/pages/guides/application_logging/splunk_cloud.md

```diff
@@ -97,4 +99,4 @@ This guide would cover configuring your app builder application to forward appli
    aio app config get log-forwarding errors
    ```
 
-   _Note: If you are unable to set up log forwarding correctly, please visit our [App Builder forums](https://experienceleaguecommunities.adobe.com/t5/project-firefly/ct-p/project-firefly) for support._
\ No newline at end of file
+   _Note: If you are unable to set up log forwarding correctly, please visit our [App Builder forums](https://experienceleaguecommunities.adobe.com/t5/project-firefly/ct-p/project-firefly) for support._
```

Comment:
the forums have changed to https://experienceleaguecommunities.adobe.com/t5/app-builder/ct-p/app-builder

Reply:
Link needs to be updated as mentioned by @shazron

Reply:
Done. Also updated in other files